### ISSUE TITLE
Deprecated strings causing test failures.

### ISSUE THREAD

**Issue by mattporritt** [moodleuulm/moodle-local_boostnavigation#15]
Hi @abias 
I was testing Release v3.5-r2 of this plugin in latest Moodle 3.5 and the following core unit test is failing:
`vendor/bin/phpunit "core_string_manager_standard_testcase" lib/tests/string_manager_standard_test.php`
It looks to be that the deprecated strings listed in lang/en/deprecated.txt have been removed from lang/en/local_boostnavigation (fully deprecated).

Do you want me to lodge a PR with the deprecated.txt file truncated or removed? Or are you happy to do it?

Cheers,
Matt P

**Comment by abias** [comment]
Hi Matt,

I'm happy to hear that you are trying this plugin :)

And I'm afraid that you are right, I have probably got the deprecating process wrong and have already removed the strings from the lang file when I deprecated them.
lang/en/deprecated.txt is now removed in 079ceb5f6eda10652567b090eca32ebac6212dd4.

Will you also need a new version in the Moodle plugin repo? Otherwise, this change will be published there with the next major changes.

Cheers,
Alex
